static/js/components/dashboard/CourseAction_test.js

```diff
@@ -41,6 +42,11 @@ describe('CourseAction', () => {
   let setEnrollCourseDialogVisibilityStub;
   let openFinancialAidCalculatorStub;
   let routerPushStub;
+  let openLinkInNewTabStub;
+  const enrollmentUrlData = [
+    { url: undefined, expected: false },
+    { url: "http://test.com", expected: true }
+  ];
```

Comment:
it looks like this array is only being used lower in the one test - can you move the declaration of it down where it is used, for readability?